package.json

```diff
@@ -1,11 +1,18 @@
 {
   "scripts": {
     "test": "xo && ava",
+    "build": "cd src && each '{*.js,!(libs)/**/*.js}' --parallel 10 -- rollup --config node:webext $PWD/%p --output $PWD/../extension/%p",
```

Comment:
It's actually more about running it with `watch` than concurrency. If `--parallel=4` and the files are 5, the 5th one won't be compiled.